src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -41,14 +42,23 @@
 #include "utils/palloc.h"
 #include "utils/relcache.h"
 
+/* expression tree walker context for rewriting row references */
+typedef struct
+{
+	char *relationName;
+	uint64 shardId;
+} RowRefWalkerState;
```

Comment:
Let's change this to `ColumnRefWalkerState` to reflect the kind of node it's actually going to modify.